### PR TITLE
Fix Search placeholder option

### LINKED ISSUE #208: Fix searchPlaceholder option
Hello,

Thank you, your plugin is awesome.
Emoji are saved in unicode in my database, so i need to display only emoji which belong  to a specific unicode version in the window.

Maximum version of unicode available need to be 4.1
https://emojipedia.org/unicode-4.1/

Currently your plugin will display "mouse emoji" but browsers will display alt="🐁" if you inspect it.
https://emojipedia.org/mouse/
I saw the function "getSupportMode" in your code but i don't know how to use it.

Another issue : 
I try to change 'searchPlaceholder' but it seems to have no effet. Indeed in emojionearea.js you have the word "SEARCH" used directly (L905) instead of the variable.

Thank you in advance!

### PR DESCRIPTION
**Pull request by carpii**
Fixes #208 
Adds demo/sandbox.html for easier testing/development

### CODE REVIEW

File: src/var/getDefaultOptions.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function($, emojione, emojioneVersion, detectVersion, getSupportMode) {
             search            : true,
             placeholder       : null,
             emojiPlaceholder  : ":smiley:",
-            searchPlaceholder : "SEARCH",
+            searchPlaceholder : null,
```
Comment (mervick) [inline src/var/getDefaultOptions.js]: Must be defined default placeholder

**Comment by carpii** [comment]
@mervick I tested it fully (on 3.x anyway). Is there an issue?

It still falls back to "SEARCH" if option is not set, and data attribute is not set either.

````
"placeholder": options.searchPlaceholder || source.data("search-placeholder") || "SEARCH",
````

But I had to remove it otherwise source.data would never work

**Comment by carpii** [comment]
Why closed? :(

**Comment by mervick** [comment]
1) `defaultOptions.js` - is the place where need to be all default values, so string "SEARCH" must be there
2) `demo/sandbox.html` - must be in separated pull request

**Pull request closed without merging**